website_product_subscription/models/res_partner.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    @api.model
+    def create_web_access(self):
+        User = self.env["res.users"]
+        for partner in self:
```

Comment:
you're looping on the self but the function annotion is model and not multi. Am I missing something ?

Reply:
nope, good point, it's fixed 👌